src/modules.js

```diff
@@ -434,10 +434,15 @@ function exportRuntime() {
     'abort',
   ];
 
+  function isJsLibraryConfigIdentifier(ident) {
```

Comment:
lgtm with also using this function in `recordGLProcAddressGet` in `library_webgl.js` and the top of `addFromLibrary` in `jsifier.js`, which also have a list like this.